gbpservice/nfp/test/unit/configurator/agent/test_input_data.py

```diff
@@ -0,0 +1,246 @@
+class FakeObjects(object):
+
+    sc = 'sc'
+    context = {'notification_data': {},
+               'resource': 'context_resource'}
+    conf = 'conf'
+    rpcmgr = 'rpcmgr'
+    nqueue = 'nqueue'
+    drivers = 'drivers'
+    vip_context = {'notification_data': {}, 'resource': 'vip'}
+    method = {'CREATE_VIP': 'create_network_device_config',
+              'DELETE_VIP': 'delete_network_device_config',
+              'UPDATE_VIP': 'update_network_device_config',
+              'CREATE_POOL': 'create_network_device_config',
+              'DELETE_POOL': 'delete_network_device_config',
+              'UPDATE_POOL': 'update_network_device_config',
+              'CREATE_MEMBER': 'create_network_device_config',
+              'DELETE_MEMBER': 'delete_network_device_config',
+              'UPDATE_MEMBER': 'update_network_device_config',
+              'CREATE_HEALTH_MONITOR': 'create_network_device_config',
+              'DELETE_HEALTH_MONITOR': 'delete_network_device_config',
+              'UPDATE_HEALTH_MONITOR': 'update_network_device_config'}
+
+    def fake_request_data_vip(self):
```

Comment:
Please follow one pattern for naming like for vip - fake_request_data_vip() is used for both create and update but for member fake_request_data_create_member() used for both create and delete. We can do this way, just have single method per resource which has common data needed for  create,update,delete. e.g fake_request_data_vip(). Based on the operation add required data by appending the key-value in the request_data['config']['kwargs']